app/services/group_service.rb

```diff
@@ -1,6 +1,6 @@
 module GroupService
-  def self.ab_test_alternative?(group)
-    group.segment_seed % 2 == 0
+  def self.ab_testing_enabled?
+    Rails.env.production? or ENV['LOOMIO_AB_TESTING_ENABLED']
```

Comment:
I'd lean towards just looking at ENV, rather than whether it's production or not. What's the likelihood a third party install is going to care about AB testing?